main.js

```diff
@@ -0,0 +1,66 @@
+
+class Person {
+  constructor(name, dateOfBirth, amountMoney) {
+  this.transaction = [];
+  this.name = name;
+  this.dateOfBirth = new Date (dateOfBirth.replace( /\./g,',').split(",").reverse().join("."));
```

Comment:
use single quotes in js,
unneeded space, need space after comma `( /\./g,',')` => `(/\./g, ',')`